src/Workspaces/Core/Portable/CodeGeneration/Symbols/CodeGenerationMethodSymbol.cs

```diff
@@ -85,6 +85,10 @@ public override ImmutableArray<ITypeSymbol> TypeArguments
 
         public override IMethodSymbol ConstructedFrom => this;
 
+        public override bool IsDeclaredReadOnly => throw new NotImplementedException();
```

Comment:
>throw new NotImplementedException(); [](start = 51, length = 36)

Minor point: If we don't expect these methods to be called, we typically `throw ExceptionUtilities.Unreachable`.